src/main/java/Duke.java

```diff
@@ -1,10 +1,221 @@
+//Matric Number: A0200273X
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        showWelcomeMessage();
+        //String Array of size 100 created to store item list
+        String[] itemList = new String[100];
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+
+        Duke numTasks =new Duke();
+        int taskCounts=0;
+        
+        /**
+         * When user enter list it will show a list of input saved
+         * to do command add a item into the list and mark it as task
+         * deadline command set a item with the deadline and mark it as deadline
+         * Event command add item as events to do and mark it as event
+         * When user enter done it will mark the task as done into the list
+         * It will run continuously till bye command is given
+         */
+        while(!line.equals("bye")){
+            for(int i=0;i<100;i++) {
+                String[] doneSide = line.split(" ");
+                String[] task = line.split(" ");
+
+                if(line.equals("list")) {
+                    System.out.print("____________________________________________________________\n");
+                    System.out.print("Here are the tasks in your list:\n");
+                    System.out.print("____________________________________________________________\n");
+
+                    for(int j=0;j<i;j++) {
+                        if(numTasks.getNumTasks()==j+1){
+                        Task t = new Task(itemList[j]);
+                        t.markAsDone();
+                            if(numTasks.getTodoNum()==j+1) {
+                                System.out.print(j + 1 + "." + "[" + "T" + "]" + "[" + t.getStatusIcon() + "] " + itemList[j] + "\n");
+
```

Comment:
Not sure if this blank line is necessary? Is it for readability? I think it would be fine without, personally.